mindsdb_sdk/agents.py

```diff
@@ -0,0 +1,188 @@
+from requests.exceptions import HTTPError
+from typing import List, Union
+import datetime
+
+from mindsdb_sdk.models import Model
+from mindsdb_sdk.skills import Skill, Skills
+from mindsdb_sdk.utils.objects_collection import CollectionBase
+
+
+class AgentCompletion:
+    """Represents a full MindsDB agent completion"""
+    def __init__(self, content: str):
+        self.content = content
+
+    def __repr__(self):
+        return self.content
+
+
+class Agent:
+    """Represents a MindsDB agent"""
+    def __init__(
+            self,
+            name: str,
+            model_name: str,
+            skills: List[Skill],
+            params: dict,
+            created_at: datetime.datetime,
+            updated_at: datetime.datetime,
+            collection: CollectionBase = None
+            ):
+        self.name = name
+        self.model_name = model_name
+        self.skills = skills
+        self.params = params
+        self.created_at = created_at
+        self.updated_at = updated_at
+        self.collection = collection
+
+    def completion(self, messages: List[dict]) -> AgentCompletion:
+        return self.collection.completion(self.name, messages)
+
+    def __repr__(self):
+        return f'{self.__class__.__name__}(name: {self.name})'
+
+    def __eq__(self, other):
+        if self.name != other.name:
+            return False
+        if self.model_name != other.model_name:
+            return False
+        if self.skills != other.skills:
+            return False
+        if self.params != other.params:
+            return False
+        if self.created_at != other.created_at:
+            return False
+        return self.updated_at == other.updated_at
+
+    @classmethod
+    def from_json(cls, json: dict, collection: CollectionBase):
+        return cls(
+            json['name'],
+            json['model_name'],
+            [Skill.from_json(skill) for skill in json['skills']],
+            json['params'],
+            json['created_at'],
+            json['updated_at'],
+            collection
+        )
+
+
+class Agents(CollectionBase):
+    """Collection for agents"""
```

Comment:
He have documentation [autogenerated](https://mindsdb.github.io/mindsdb_python_sdk/) from docstrings. 
Would be great to have here (and in Skills class) some brief info how to use it. 
Examples from other classes [1](https://github.com/mindsdb/mindsdb_python_sdk/blob/628a7c7d1494ac7e045472a4e08ad90eafd2ed09/mindsdb_sdk/views.py#L38), [2](https://github.com/mindsdb/mindsdb_python_sdk/blob/06202aea36d67823c53b16f979e40dfd711e5439/mindsdb_sdk/tables.py#L179)